paddle/fluid/operators/center_loss_op.cc

```diff
@@ -141,6 +141,9 @@ class CenterLossOpGradMaker : public framework::SingleGradOpMaker<T> {
     return retv;
   }
 };
+
+DECLARE_NO_NEED_BUFFER_VARS_INFERENCE(CenterLossGradNoNeedBufVarsInferer, "X");
```

Comment:
Seems that the CUDA kernel of `center_loss_grad` would use the buffer of `X`, see `center_loss_op.cu` to check it? Maybe we should change the implementation of the CUDA kernel.

Reply:
Thanks for finding this, I will add center_loss_grad to the white_list and request OP owner to change it later.

Reply:
It seems that OpTest cannot check the correctness of `DECLARE_NO_NEED_BUFFER_VARS_INFERENCE`? Maybe we need to enhance that?

Reply:
My fault. This changing is right.